core/src/main/resources/init/init.ftl

```diff
@@ -18,4 +22,4 @@ ${customUserData}
 
 find /usr/lib -name "os_family.json*" | xargs sed -i 's/amazon2015/amazon2016/g'
 
-/usr/bin/user-data-helper.sh "$@" &> /var/log/user-data.log
\ No newline at end of file
+/usr/bin/user-data-helper.sh "$@" &> /var/log/user-data.log
```

Comment:
If the 4.line already redirects all aoutput to `user-data.log`, than the seconf half of this line `&> /var/log/user-data.log` is really needed?